src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {person.name}
+    </h2>
+
+    {person.age
+      ? (
+        <p className="Person__age">
+          I am
+          {' '}
+          {person.age}
+        </p>
+      ) : null}
```

Comment:
use interpolation.
when you need to render smth only if condition is true it is better to use And (&&)  logical operator instead of ternary with the second option as null.

```javascript
{perosn.age && (
// your jsx goes here
)
```